Libraries/LibWeb/CSS/StyleValues/ColorSchemeStyleValue.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2025, Ladybird contributors
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "ColorSchemeStyleValue.h"
+#include <LibWeb/CSS/Serialize.h>
+
+namespace Web::CSS {
+
+String ColorSchemeStyleValue::to_string(SerializationMode) const
+{
+    if (schemes().is_empty())
+        return "normal"_string;
+
+    StringBuilder builder;
+    bool first = true;
+    for (auto const& scheme : schemes()) {
+        if (first) {
+            first = false;
+        } else {
+            builder.append(' ');
+        }
+        builder.append(scheme);
+    }
```

Comment:
I was going to say you could use `builder.join(' ', schemes());` here... but actually this needs to make use of `serialize_an_identifier()` I think, because a scheme could be a `<custom-ident>`. That function escapes any funky characters.

Reply:
Ah, forgot about that, done :+1: